moveit_setup_assistant/src/widgets/setup_assistant_widget.cpp

```diff
@@ -113,13 +113,7 @@ SetupAssistantWidget::SetupAssistantWidget(QWidget* parent, const boost::program
   }
   else
   {
-    // Open the directory where the MSA was started from.
-    // cf. http://stackoverflow.com/a/7413516/577001
-    QString pwdir("");
-    char* pwd;
-    pwd = getenv("PWD");
-    pwdir.append(pwd);
-    start_screen_widget_->stack_path_->setPath(pwdir);
+    start_screen_widget_->stack_path_->setPath(QString(getenv("PWD")));
```

Comment:
this looks different here: https://github.com/ros-planning/moveit/pull/1998/commits/eb02dcdba7ac9312202b3bdd56b94006e7f7bd17    ie it uses QDir::currentPath() ?

Reply:
It turned out that `QDir::currentPath()` doesn't return the original working directory (see #1932).